WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -446,9 +446,16 @@ && getIntent().getExtras().getBoolean(ARG_CONTINUE_JETPACK_CONNECT, false)) {
             onSetPromptReminderClick(getIntent().getIntExtra(ARG_OPEN_BLOGGING_REMINDERS, 0));
         }
 
+        int selectedSiteLocalId = mSelectedSiteRepository.getSelectedSiteLocalId(true);
+        SiteModel site = selectedSiteLocalId == SelectedSiteRepository.UNAVAILABLE ? null
+                : mSiteStore.getSiteByLocalId(selectedSiteLocalId);
+
         if (BuildConfig.IS_JETPACK_APP
             && mStatsRevampV2FeatureConfig.isEnabled()
-            && AppPrefs.shouldDisplayStatsRevampFeatureAnnouncement()) {
+            && AppPrefs.shouldDisplayStatsRevampFeatureAnnouncement()
+            && site != null
+            && site.isUsingWpComRestApi()
```

Comment:
Since we're already checking IS_JETPACK_APP, isUsingWpComRestApi() check  seems redundant here

Reply:
@frosty said we should check if the site is a Jetpack site, and he did the same here, [StatsRevampV2IntroductionPresenter.swift](https://github.com/wordpress-mobile/WordPress-iOS/blob/079e26c1284568555eae5da1bf6064d56298857f/WordPress/Classes/ViewRelated/Feature%20Introduction/Stats%20Revamp%20v2/StatsRevampV2IntroductionPresenter.swift#L66-L74).
But you're right, `IS_JETPACK_APP` should be doing the same check.

Reply:
Removed b3044b1be1302bcbea3bc2cac90cc657b5da1435